mawabot/cogs/text.py

```diff
@@ -82,6 +83,36 @@ def __init__(self, bot):
 
         await ctx.message.edit(content='\n'.join(lines))
 
+    @commands.command()
+    async def rev(self, ctx, *, text: str):
+        ''' Reverses the text given '''
+
+        await ctx.message.edit(content=text[::-1])
+
+    @commands.command()
+    async def kerrhau(self, ctx, *text: str):
+        ''' "kerrhau"-ifys the given text for meme emphasis '''
+
+        text = list(text)
+        words = []
+
+        while text:
+            rand_num = randint(1, 3)
+            word = []
+
+            for _ in range(rand_num):
+                if text:
+                    word.append(text.pop(0))
+
+            words.append(' '.join(word))
+
+        last = words[-1][-1]
+        words[-1] = words[-1][:-1]
+        words.append(last)
+
+        await ctx.message.edit(content='\n'.join(words))
+
+
```

Comment:
I don't think we're using the double blank line for function thing in the rest of the project.
I might be wrong though,